### PR TITLE
c2c: skip TestStreamingAutoReplan

### PR DESCRIPTION
**Pull request by msbutler**
This patch also switches AutoReplanning default off.

Informs #106451

Release note: None

### CODE REVIEW

File: pkg/ccl/streamingccl/streamingest/stream_ingestion_dist.go [hunk 1]
```diff
@@ -40,7 +40,7 @@ var replanThreshold = settings.RegisterFloatSetting(
 	"stream_replication.replan_flow_threshold",
 	"fraction of nodes in the producer or consumer job that would need to change to refresh the"+
 		" physical execution plan. If set to 0, the physical plan will not automatically refresh.",
-	0.1,
+	0,
```
Comment (renatolabs) [inline pkg/ccl/streamingccl/streamingest/stream_ingestion_dist.go:43]: This seems unrelated.

Comment (msbutler) [reply pkg/ccl/streamingccl/streamingest/stream_ingestion_dist.go:43]: i'd like to turn the feature default off, as the patch skips the only e2e test that exercises this feature.

**Review by renatolabs — APPROVED** [review]

**Comment by msbutler** [comment]
bors p=999
